templates/includes/contact-us.html

```diff
@@ -10,141 +10,227 @@ <h2 class="p-modal__title" id="modal-title">Get your Kubeflow stack running</h2>
+              <span class="p-checkbox__label">Oracle Cloud</span>
+            </label>
+            <div class="js-other-container" style="margin-top: -.45rem;">
+              <label class="p-checkbox" for="workloads-public-cloud-other">
+                <input type="checkbox" class="js-other-container__checkbox p-checkbox__input" id="workloads-public-cloud-other">
+                <span class="p-checkbox__label">Other</span>
+              </label>
+              <label for="workloads-public-cloud-other-specified">
+                <input type="text" id="workloads-public-cloud-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0;" aria-label="Other private cloud platform">
+              </label>
             </div>
           </div>
           <div class="col-4 u-sv3">
-            <input type="checkbox" id="workloads-metal-Nvidia">
-            <label for="workloads-metal-Nvidia">Nvidia</label>
-            <input type="checkbox" id="workloads-metal-AMD">
-            <label for="workloads-metal-AMD">AMD</label>
-            <input type="checkbox" id="workloads-metal-Intel">
-            <label for="workloads-metal-Intel">Intel</label>
-            <input type="checkbox" id="workloads-metal-POWER">
-            <label for="workloads-metal-POWER">POWER</label>
-            <input type="checkbox" id="workloads-metal-Z">
-            <label for="workloads-metal-Z">Z</label>
+            <label class="p-checkbox" for="workloads-metal-Nvidia">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-metal-Nvidia">
+              <span class="p-checkbox__label">Nvidia</span>
+            </label>
+            <label class="p-checkbox" for="workloads-metal-AMD">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-metal-AMD">
+              <span class="p-checkbox__label">AMD</span>
+            </label>
+            <label class="p-checkbox" for="workloads-metal-Intel">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-metal-Intel">
+              <span class="p-checkbox__label">Intel</span>
+            </label>
+            <label class="p-checkbox" for="workloads-metal-POWER">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-metal-POWER">
+              <span class="p-checkbox__label">POWER</span>
+            </label>
+            <label class="p-checkbox" for="workloads-metal-Z">
+              <input class="p-checkbox__input" type="checkbox" id="workloads-metal-Z">
+              <span class="p-checkbox__label">Z</span>
+            </label>
           </div>
         </div>
       </div>
-      <div class="pagination">
-        <a class="p-button--positive pagination__link--next" href="">Next</a>
-      </div>
+      <ol class="p-pagination">
+        <li class="p-pagination__item">
+          <a class="p-button--positive p-pagination__link--next" href="">Next</a>
```

Comment:
```suggestion
          <a class="p-button--positive" href="">Next</a>
```

Reply:
mm looks like the pagination styling is overriding the button color. The `pagination__link--next` is required for the script to work, so I can't remove it. I'll add hardcode the styles.